.github/actions/test-and-build/action.yml

```diff
@@ -16,31 +16,32 @@ inputs:
 runs:
   using: 'composite'
   steps:
-    - uses: actions/setup-node@v3
+    - name: 'Install PNPM'
+      uses: pnpm/action-setup@ebcfd6995dade4b0104ac774445cef8b3b4635b0
       with:
-        node-version: '18'
+        version: 8
     - name: Cache
```

Comment:
let's just install node with `pnpm` cache instead of this action, could be faster as I see (https://github.com/fingerprintjs/action-coverage-report-md/actions/runs/8239584214/job/22533129323)

Reply:
Done: https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/pull/88/commits/597fdf434af8c7bb25eaf6ea3bd6ceeb1de98126